1.8/usage/jobs/examples.md

```diff
@@ -5,56 +5,94 @@ feature_maturity: preview
 menu_order: 20
 ---
 
-# A Simple Job
-
-```json
-{
-  "id": "my-job",
-  "description": "A job that sleeps",
-  "run": {
-    "cmd": "sleep 1000",
-    "cpus": 0.01,
-    "mem": 32,
-    "disk": 0
-  }
-}
-```
-
-# A Job with a Schedule
-
-```
-{
-  "id": "my-scheduled-job",
-  "description": "A job that sleeps on a schedule",
-  "run": {
-    "cmd": "sleep 20000",
-    "cpus": 0.01,
-    "mem": 32,
-    "disk": 0
-  },
-  "schedules": [
+<a name="create-job"></a>
+# Create a Simple Job
+
+This JSON creates a simple job with no schedule.
+
+1. Create a JSON file with the following contents. 
+    ```json
     {
-      "id": "sleep-nightly",
-      "enabled": true,
-      "cron": "20 0 * * *",
-      "concurrencyPolicy": "ALLOW"
+      "id": "my-job",
+      "description": "A job that sleeps",
+      "run": {
+        "cmd": "sleep 1000",
+        "cpus": 0.01,
+        "mem": 32,
+        "disk": 0
+      }
     }
-  ]
-}
-```
-
-# A Schedule-Only JSON file
-
-```
-[
-  {
-    "concurrencyPolicy": "ALLOW",
-    "cron": "20 0 * * *",
-    "enabled": true,
-    "id": "nightly",
-    "nextRunAt": "2016-07-26T00:20:00.000+0000",
-    "startingDeadlineSeconds": 900,
-    "timezone": "UTC"
-  }
-]
-```
+    ```
+
+1. Add the job from the DC/OS CLI.
+    ```bash
+    dcos job add <my-job>.json
+    ```
+    
+    Alternatively, add the job using the API.
+    ```bash
+    curl -X POST -H "Content-Type: application/json" -H "Authorization: token=$(dcos config show core.dcos_acs_token)" $(dcos config show core.dcos_url)/service/metronome/v1/jobs -d@/Users/<your-username>/<myjob>.json
+    ```
+
+# Create a Job with a Schedule via the DC/OS CLI
+**Note:** This example JSON only works when you add the job from the DC/OS CLI. Use the [example below](#schedule-with-api) to create a scheduled job via the API.
```

Comment:
the claim is this is only possible with the CLI however it looks like the example below for the UI is the same.  the restriction is that it can NOT be used directly via the job end point.   The UI and CLI will make multiple http requests to make this json convenient.